### PR TITLE
Update readme for PlayFab Multiplayer Unreal OSS

### PR DESCRIPTION
**Pull request by Jeffrey-Shi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
...
+
+Preview SDKs and engine plugins might not support all the scenarios planned for the  generally available (GA) release and are not necessarily fully compliant with the certification requirements for the target platform.
+
+For platform certification and shipping to retail users, games must use the generally available (GA) release of the SDKs and engine plugins that will become available in the coming months.
+
+For games shipping within the next 3 months to Xbox console and PC Game Pass program, it is recommended that games use the base Online Subsystem (OSS) for GDK provided by Epic for multiplayer integration (backed by Xbox Live multiplayer services) and the PlayFab Online Subsystem (PF OSS) v1.x for Party networking and Voice over Internet Protocol (VOIP) available on the [GDK download website](https://aka.ms/gdkdl).
+
+## Known Limitations and Issues in the Current Preview Build
+
+- [General] Currently cross-play is only supported between Xbox consoles and PC Game Pass.  Cross-play support between other platforms is coming soon.
```
Comment (debhaldarMS) [inline README.md:49]: Cross play is also supported between GDK<-> Steam<-Switch

File: README.md [hunk 2]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
 keywords: playfab, multiplayer, networking, unreal, unreal engine, unreal engine 4, middleware
 ---
 
-# Getting Started
+## Introduction
+
+The PlayFab Online Subsystem (PF OSS) v2 enables you to make use of PlayFab Multiplayer features like Lobby, Matchmaking, Party and Azure Cognitive Services in your Unreal Engine 4 (UE4) game.
```
Comment (joannaleecy) [inline README.md:14]: What if I'm using UE5?

File: README.md [hunk 3]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
...
+
+These features include cross-platform VOIP , cross-play, and accessibility features like real-time text chat translation and voice transcription services. It is currently designed for use when developing PC, Xbox, Steam and Nintendo Switch games. This OSS layer works seamlessly on top of the existing base platform OSS layers (such as the base GDK OSS layer provided by Epic for Xbox and PC Game Pass). PlayFab OSS compliments the base OSS by adding support for PlayFab Lobby, Matchmaking, Party networking and Voice over Internet Protocol (VOIP).
```
Comment (joannaleecy) [inline README.md:16]: Does it not work for PS titles?

Comment (joannaleecy) [inline README.md:16]: Would be good to provide a link to the base OSS, so they can go to it easily from here.

File: README.md [hunk 4]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
 keywords: playfab, multiplayer, networking, unreal, unreal engine, unreal engine 4, middleware
 ---
 
-# Getting Started
```
Comment (joannaleecy) [inline README.md:12]: If this were a page in the docs, you need at least one H1 header. (aka. at least one single #)

File: README.md [hunk 5]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
...
+Preview SDKs and engine plugins might not support all the scenarios planned for the  generally available (GA) release and are not necessarily fully compliant with the certification requirements for the target platform.
+
+For platform certification and shipping to retail users, games must use the generally available (GA) release of the SDKs and engine plugins that will become available in the coming months.
+
+For games shipping within the next 3 months to Xbox console and PC Game Pass program, it is recommended that games use the base Online Subsystem (OSS) for GDK provided by Epic for multiplayer integration (backed by Xbox Live multiplayer services) and the PlayFab Online Subsystem (PF OSS) v1.x for Party networking and Voice over Internet Protocol (VOIP) available on the [GDK download website](https://aka.ms/gdkdl).
+
+## Known Limitations and Issues in the Current Preview Build
+
+- [General] Currently cross-play is only supported between Xbox consoles and PC Game Pass.  Cross-play support between other platforms is coming soon.
+- [Xbox] Currently cannot use Xbox shell to send or receive platform level invites with PF OSS v2.  In-game invite works.
```
Comment (joannaleecy) [inline README.md:50]: Would be good to know if these functionalities will become available in public preview or GA.

File: README.md (hunk 3 above)
Comment (PFarqMS) [reply README.md:16]: Hard to link to the OSS, but we could link to the docs page we have about how to configure this to use the base OSS.

File: README.md (hunk 4 above)
Comment (Jeffrey-Shi) [reply README.md:12]: This is for the GitHub repo landing page only, we are not adding the same content to the doc.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jeffrey-Shi) [reply README.md]: Changed to cross-network-play

File: README.md [hunk 6]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
...
+
+## Important Note
+
+The PlayFab Online Subsystem (PF OSS) v2 is currently in preview. Preview programs give Microsoft and customers the opportunity to evaluate and understand the new feature before it becomes part of the standard feature set of PlayFab.
+
+Preview SDKs and engine plugins might not support all the scenarios planned for the  generally available (GA) release and are not necessarily fully compliant with the certification requirements for the target platform.
+
+For platform certification and shipping to retail users, games must use the generally available (GA) release of the SDKs and engine plugins that will become available in the coming months.
+
+For games shipping within the next 3 months to Xbox console and PC Game Pass program, it is recommended that games use the base Online Subsystem (OSS) for GDK provided by Epic for multiplayer integration (backed by Xbox Live multiplayer services) and the PlayFab Online Subsystem (PF OSS) v1.x for Party networking and Voice over Internet Protocol (VOIP) available on the [GDK download website](https://aka.ms/gdkdl).
```
[the remark this answers is not in the document]
Comment (Jeffrey-Shi) [reply README.md]: Changed to 'before September 2022'

File: README.md (hunk 2 above)
Comment (Jeffrey-Shi) [reply README.md]: Changed to 'Unreal Engine game'.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jeffrey-Shi) [reply README.md]: Updated according to comment.

File: README.md [hunk 7]
```diff
@@ -9,20 +9,43 @@ ms.prod: playfab
...
+
+The PlayFab Online Subsystem (PF OSS) v2 enables you to make use of PlayFab Multiplayer features like Lobby, Matchmaking, Party and Azure Cognitive Services in your Unreal Engine 4 (UE4) game.
+
+These features include cross-platform VOIP , cross-play, and accessibility features like real-time text chat translation and voice transcription services. It is currently designed for use when developing PC, Xbox, Steam and Nintendo Switch games. This OSS layer works seamlessly on top of the existing base platform OSS layers (such as the base GDK OSS layer provided by Epic for Xbox and PC Game Pass). PlayFab OSS compliments the base OSS by adding support for PlayFab Lobby, Matchmaking, Party networking and Voice over Internet Protocol (VOIP).
+
+## Getting Started
 
 The quickstart guide for OnlineSubsystemPlayFab can be found at:
 
 [PlayFab Online Subsystem](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/networking/party-unreal-engine-oss-quickstart)
```
[the remark this answers is not in the document]
Comment (Jeffrey-Shi) [reply README.md]: Done.

File: README.md (hunk 5 above)
Comment (Jeffrey-Shi) [reply README.md]: Added note saying it would be supported in GA release.

File: README.md (hunk 3 above)
Comment (Jeffrey-Shi) [reply README.md]: Not yet - see below for supported platforms list.  Will update this section once we have PS support.

File: README.md (hunk 1 above)
Comment (Jeffrey-Shi) [reply README.md]: Updated.

File: README.md (hunk 3 above)
Comment (Jeffrey-Shi) [reply README.md]: I can only find the link to how to setup GDK OSS for PC.  I think the how-to for Xbox is on Epic secure dev site so not sure we want to include a link here.  Also for devs shipping to Xbox and PC Game Pass very likely they learn about the GDK OOS already before getting into this section, so I think this is more a nice to have change.

I'd like to push out the PR asap to make sure people see the important note - if we can find a good link to the base OSS we can always update this readme.

**Review by PushpadantK — APPROVED** [review]
:shipit:

**Pull request merged**